ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -1626,6 +1626,17 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-ipi-ovn-ipsec-azurefile-csi
+- as: azure-ipi-ovn-workload-identity-p1-f3
```

Comment:
`f3` ? I think it is to aggressive, this feature sounds like a TP feature, I think `f14`  is enough.

Reply:
And if it is TP feature, pls add `tp` keyword in the job name, also pls remove `ovn` keyword from its name.

Reply:
Also need to add the similar job in arm nightly ci.

Reply:
@jianlinliu  Thank you very much, I agree with your opinion, but at the beginning we need to debug to make this ci reach a relatively stable state. After stabilization, the frequency will be adjusted to f14 or f28. 
If f3 is to aggressive, what about f7?

Reply:
Now we can trigger any job via remote api, https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/README.md#trigger-a-new-test-for-an-existing-periodic-job-in-prow, so I do not think it is a problem for your debugging. You can trigger it any time on your demand.

And from the whole team management review, we add many jobs every day, if each team think it is important to have a ci resulta in short days, then our quota would be easily run out, unless there is a strong reason to set an aggressive test frequency. 

If we set f14 for both arm and amd, then the job would be executed 1 time in 1 week on average, I think it is reasonable enough for TP feature job.

Reply:
@jianlinliu Thanks for your information!! If so, agree to tune to f14.
 I can use this https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/README.md#trigger-a-new-test-for-an-existing-periodic-job-in-prow

Reply:
Hi @jianlinliu, arm nightly ci add down. Could you help review again? Thanks.